test/providers/index.test.js

```diff
@@ -20,6 +20,7 @@ describe('CI Providers', () => {
         GITHUB_ACTIONS: true,
         GITHUB_REF: 'refs/heads/test',
         GITHUB_REPOSITORY: 'testOrg/testRepo',
+        JENKINS_URL: 'https://example.om',
```

Comment:
Not really an issue, but did you mean `com`?

Reply:
```suggestion
        JENKINS_URL: 'https://example.com',
```

Reply:
whoops I did